openlattice.yaml

```diff
@@ -2981,18 +3235,20 @@ components:
               type: string
 
 tags:
-- name: edm
-  description: API endpoints to the entity data model.
-- name: permissions
-  description: API endpoints for permissions for organisations, roles and users.
-- name: directory
-  description: API endpoints for searching users and their properties.
-- name: data
-  description: API endpoints to reading and writing data.
-- name: search
-  description: API endpoints to search data.
-- name: organizations
-  description: API endpoints related to organizations.
+  - name: edm
+    description: API endpoints to the entity data model.
+  - name: permissions
+    description: API endpoints for permissions for organisations, roles and users.
```

Comment:
I like Europe but we should probably be consistent with spellings for docs :) (and probably better to stick with the spellings that are used in api names/paths)

Reply:
Lol, so much time I've spent changing z vs s.
There were a few of these in the code too (`MATERIALIZED` vs `MATERIALISED` 🤦‍♀ ), the latest commit took those out...